index.html

```diff
@@ -1732,15 +1744,23 @@ <h2>
           Processing shortcut items
         </h2>
         <p>
-          To <dfn>process a shortcut</dfn>, given [=object=] |item:object|:
+          To <dfn>process a shortcut</dfn>, given [=ordered map=] |item:ordered
+          map|:
         </p>
         <ol class="algorithm">
-          <li>If the type of |item| is not [=object=], return failure.
-          </li>
-          <li>If |item|["name"] is missing or |item|["name"] is the empty
-          string, return failure.
-          </li>
-          <li>If the type of |item|["url"] is not [=string=], return failure.
+          <li>Return failure if it's the case that:
+            <ul>
+              <li>|item| is not [=ordered map=].
+              </li>
+              <li>|item|["name"] doesn't [=map/exist=].
+              </li>
+              <li>|item|["name"] is the empty string.
```

Comment:
isn't there a reference for empty string?

Reply:
Not for string, it seems: https://respec.org/xref/?term=empty